grafana-overview-dashboard.md

```diff
@@ -19,59 +19,56 @@ To understand the key metrics displayed on the Overview dashboard, check the fol
-System Info | Vcores | the number of CPU cores |
-System Info | Memory | the total memory |
-System Info | CPU Usage | the CPU usage ratio, 100% at a maximum |
-System Info | Load [1m] | the overload within 1 minute |
-System Info | Memory Available | the size of the available memory |
-System Info | Network Traffic | the statistics of the network traffic |
-System Info | TCP Retrans | the statistics about network monitoring and TCP |
-System Info | IO Util | the disk usage ratio, 100% at a maximum; generally you need to consider adding a new node when the usage ratio is up to 80% ~ 90% |
+Services Port Status | Services Up | The online nodes number of each service. |
+PD | PD role | The role of the current PD. |
+PD | Storage capacity |  The total storage capacity of the TiDB cluster. |
+PD | Current storage size | The occupied storage capacity of the TiDB cluster, including the space occupied by TiKV replicas. |
+PD | Normal stores | The number of nodes in the normal state. |
+PD | Abnormal stores | The number of nodes in the abnormal state. | 0 |
+PD | Number of Regions | The total number of Regions in the current cluster. Note that the number of Regions has nothing to do with the number of replicas. |
+PD | 99% completed_cmds_duration_seconds | The 99th percentile duration to complete a pd-server request. | less than 5ms
+PD | Handle\_requests\_duration\_seconds | The network duration of a PD request. |
+PD | Region health | The state of each Region. | Generally, the number of pending peers is less than 100, and that of the miss peers cannot always be greater than 0. |
+PD | Hot write Region's leader distribution | The total number of leader Regions in hot write on each TiKV instance. |
+PD | Hot read Region's leader distribution | The total number of leader Regions in hot read on each TiKV instance. |
+PD | Region heartbeat report | The count of heartbeats reported to PD per instance. |
+PD | 99% Region heartbeat latency | The heartbeat latency per TiKV instance (P99). |
+TiDB | Statement OPS | The number of different types of SQL statements executed per second, which is counted according to `SELECT`, `INSERT`, `UPDATE`, and other types of statements. |
+TiDB | Duration | The execution time.<br/>1. The duration between the time that the client's network request is sent to TiDB and the time that the request is returned to the client after TiDB has executed it. In general, client requests are sent in the form of SQL statements; however, this duration can include the execution time of commands such as `COM_PING`, `COM_SLEEP`, `COM_STMT_FETCH`, and `COM_SEND_LONG_DATA`.<br/>2. Because TiDB supports Multi-Query, it supports sending multiple SQL statements at one time, such as `select 1; select 1; select 1;`. In this case, the total execution time of this query includes the execution time of all SQL statements. |
```

Comment:
```suggestion
TiDB | Duration | The execution time.<br/>1. The duration between the time that the client's network request is sent to TiDB and the time that the request is returned to the client after TiDB has executed the request. In general, client requests are sent in the form of SQL statements; however, this duration can include the execution time of commands such as `COM_PING`, `COM_SLEEP`, `COM_STMT_FETCH`, and `COM_SEND_LONG_DATA`.<br/>2. Because TiDB supports Multi-Query, TiDB supports sending multiple SQL statements at one time, such as `select 1; select 1; select 1;`. In this case, the total execution time of this query includes the execution time of all SQL statements. |
```